src/components/Observations/ObsListToolbar.js

```diff
@@ -80,15 +97,17 @@ const Toolbar = ( { setLayout, layout }: Props ): Node => {
   /* eslint-disable react-native/no-inline-styles */
   return (
     <View className="bg-white border-b border-[#e8e8e8]">
-      <View className="py-5 flex flex-row items-center px-[15px]">
+      <View className="pt-5 flex flex-row items-center px-[15px]">
         {currentUser && (
-          <Pressable
-            className="mr-3"
-            accessibilityRole="button"
+          <IconButton
+            icon="compass-rose"
             onPress={( ) => navigation.navigate( "MainStack", { screen: "Explore" } )}
-          >
-            <IconMaterial name="language" size={30} />
-          </Pressable>
+            accessibilityLabel={t( "Explore" )}
```

Comment:
I have changed the accessibility label here into a hint. What we used to write as labels, the longer half sentence like strings, are actually more to be used as hints according to the iOS documentation at least. The labels are supposed to be succinct, one or two words. We need to refactor those into hints everywhere we used them.
I added some more explanation in the string.ftl file: [704](https://github.com/Chrischuck/iNaturalistReactNative/blob/b0b25e4dd4b0cbd9277e0d47703f894f14e37fb5/src/i18n/strings.ftl#L704-L705)

Reply:
Sounds good! will make that a note for the future